config/initializers/decidim.rb

```diff
@@ -14,7 +14,7 @@
   # of languages will be equal or a subset of the list in this file.
   # config.available_locales = Rails.application.secrets.decidim[:available_locales].presence || [:en]
   # Or block set it up manually and prevent ENV manipulation:
-  config.available_locales = %w(en ca es)
+  config.available_locales = %w(en it bg el)
```

Comment:
I don´t like this solution. I prefer to receive this value from the secrets and define the values in environment variables in the server.

```suggestion
  config.available_locales = Rails.application.secrets.decidim[:available_locales].presence || [:en]
```

And in the server include the ENV variable `DECIDIM_AVAILABLE_LOCALES: en,it,bg,el`